geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/StartLocatorCommandTest.java

```diff
@@ -246,7 +246,6 @@ public void testCreateStartLocatorCommandLineWithAllOptions() throws Exception {
 
     assertNotNull(commandLineElements);
     assertTrue(commandLineElements.length > 0);
-    assertEquals(commandLineElements.length, expectedCommandLineElements.size());
```

Comment:
why are we removing the assertions? if you use assertJ's assertThat(commandLineElements).isSameAs(expectedCommandLineElements), it should compare the elements of the list (if it's a list) and if not, it will print out the differences for you.

Reply:
Good suggestion.